cmd/codegen/generator/typescript/templates/src/method_solve.ts.gtpl

```diff
@@ -81,8 +81,9 @@
 
       		{{- with $optionals }}
       			{{- if $required }}, {{ end }}
-				{{- "" }}...opts{{- if gt (len $enums) 0 -}}, __metadata: metadata{{- end -}}
+				{{- "" }}...opts
 			{{- end }}
+      {{- if gt (len $enums) 0 -}}, __metadata: metadata{{- end -}}
```

Comment:
I think you'll need to regenate the TS client if you updated the generator

Reply:
> While this isn't used by any API at the moment

This isn't currently used by any core API at the moment, so the codegen doesn't need to be run. The linter detects if it's not been run, and it's not currently failing, so it's up to date.

Reply:
Okay, LGTM then